### ISSUE TITLE
set(getvariableofpc(<var>, <account id>), <val>) crashes map server

### ISSUE THREAD

**Issue by EyesOfAHawk** [HerculesWS/Hercules#2212]
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Issue Prelude

[//]: # (Please complete these mandatory steps and check the following boxes by putting an `x` inside the brackets _before_ filing your issue)

-  [ ] I have not modified the source prior to reproducing this issue.
-  [x] I am using the latest version of Hercules.
-  [x] I am aware that this report will be closed or deleted if it becomes obvious that I am stating the false.

I am using Gepard Shield.

### Description

A script I made crashes the server. It involved `set(getvariableofpc())`.

[//]: # (Tell us what should happen instead.)

### Steps To Reproduce The Issue

1. I used these commands to create a mute log for a player. Already checked player is logged in earlier in script.
```
set(getvariableofpc(#Mute_Log[0], .@tsd), .@Amount + 1);
set(getvariableofpc(#Mute_Log[1], .@tsd), .@Original + .@Time);
```
2. Once script is done executing, log out. The server will crash with the following log/reference (using gdb).
https://pastebin.com/dcRmP7Jd

I can send my full script to somebody if they want to test.

**Branch(es):**
- [x] master
- [x] stable

### Operating System

CentOS7.4

**Comment by 4144** [comment]
``.@tsd`` is loggedin player or offline?

**Comment by EyesOfAHawk** [comment]
Yep I have checked player is logged in before executing.

**Comment by Helianthella** [comment]
@EyesOfAHawk thank you for this bug report, it is fixed in #2238
